client/components/common/SelectorBox.jsx

```diff
@@ -143,7 +143,7 @@ function Collapse({ children }) {
 
   return (
     <CollapseMUI in={expanded}>
-      <CardContent className={classes.content}>{children}</CardContent>
+      <Box className={classes.content}>{children}</Box>
```

Comment:
What was the reason for changing it to Box component?

Reply:
As far as I can tell, all the `CardContent` component does is add an unwanted 24px padding on the bottom. [CardContent source code here. ](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/CardContent/CardContent.js) Replacing it with `Box` seemed better than removing the bottom padding from `CardContent` (because then what's the point of even using it) or having to add a negative bottom margin to whatever is nested in `CardContent`. 

You can see that unwanted padding in the datepicker dropdown.

![Screen Shot 2021-01-22 at 9 16 57 AM](https://user-images.githubusercontent.com/40484278/105522884-a3526a00-5c92-11eb-9008-6878363421b6.png)